lib/cylc/task_pool.py

```diff
@@ -1282,18 +1290,18 @@ def process_queued_db_ops(self):
 
     def force_spawn(self, itask):
         """Spawn successor of itask."""
-        if itask.state.has_spawned():
+        if itask.has_spawned:
             return None
-        itask.state.set_spawned()
+        itask.has_spawned = True
         itask.log(DEBUG, 'forced spawning')
-        new_task = itask.spawn('waiting')
+        new_task = itask.spawn(TASK_STATUS_WAITING)
         if new_task and self.add_to_runahead_pool(new_task):
             return new_task
         else:
             return None
 
-    def spawn_tasks(self):
-        """Spawn successors of tasks in pool.
+    def spawn_all_tasks(self):
+        """Spawn successors of tasks in pool, if they're ready.
```

Comment:
Does this change spawning behaviour or is it more of a refactor of existing code with same end behaviour? (can't really tell from reading through)